lib/fastlane/plugin/ddg_apple_automation/helper/ddg_apple_automation_helper.rb

```diff
@@ -36,6 +36,14 @@ class DdgAppleAutomationHelper
                            ])
       }.freeze
 
+      def self.get_release_branch_name(platform, version)
+        "#{RELEASE_BRANCH}/#{platform}/#{version}"
+      end
+
+      def self.get_hotfix_branch_name(platform, version)
+        "#{HOTFIX_BRANCH}/#{platform}/#{version}"
+      end
```

Comment:
Just to keep naming consistent, could these functions be called `release_branch_name` and `hotfix_branch_name`? Just like we have `current_version` and `current_build_number` defined elsewhere in this file.